### PR TITLE
Dev/add secure rom bl host example

### PR DESCRIPTION
**Pull request by ozersa**
MSDK-975:
Added secure ROM bootloader host example on MAX32660/MAX32670/MAX32570 MCU
This example demonstrate how to communication with secure ROM BL

MAX32660/MAX32670/MAX32570 FW tested with 
- MAX32520-EvKIT
- MAX32651 EvKIT

Related [PR#7](https://github.com/Analog-Devices-MSDK/Examples-MAX32570/pull/7)

Signed-off-by: Sadik.Ozer [sadik.ozer@analog.com](mailto:sadik.ozer@analog.com)

### CODE REVIEW

**Review by Jake-Carter — APPROVED** [review]
Looks good!

File: Examples/MAX32670/SecureROM_BL_Host/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+## Description
+
+The Analog Devices Inc. (ADI) secure microcontroller has special bootloader, it is called as "Secure ROM Bootloader"
+this bootloaders provide firmware update function by secure way. The security related staff is handle by SDK and SDK tools.
```
Comment (sihyung-maxim) [inline Examples/MAX32670/SecureROM_BL_Host/README.md:4]: Period and capitalization typo, each line is a sentence.

File: Examples/MAX32660/SecureROM_BL_Host/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+## Description
+
+The Analog Devices Inc. (ADI) secure microcontroller has special bootloader, it is called as "Secure ROM Bootloader"
+this bootloaders provide firmware update function by secure way. The security related staff is handle by SDK and SDK tools.
```
Comment (sihyung-maxim) [inline Examples/MAX32660/SecureROM_BL_Host/README.md:4]: Period and capitalization typo, each line is a sentence.

**Review by sihyung-maxim — APPROVED** [review]
Looks good. Just found a minor typo.

**Comment by sihyung-maxim** [comment]
I remember around early December or so, the "BLE Examples Test" action was getting updated. I'm not sure what the status of that action is now.

**Comment by EdwinFairchild** [comment]
merge main into this and it should get the recently merged ble test

File: Examples/MAX32670/SecureROM_BL_Host/README.md (hunk 1 above)
Comment (ozersa) [reply Examples/MAX32670/SecureROM_BL_Host/README.md]: Thanks updated.

File: Examples/MAX32660/SecureROM_BL_Host/README.md (hunk 2 above)
Comment (ozersa) [reply Examples/MAX32660/SecureROM_BL_Host/README.md]: Thanks updated.

**Comment by ozersa** [comment]
@Jake-Carter  @sihyung-maxim  may it to be merged (if there is not any open item)?

**Pull request merged**